Fix rpath problem

snoweye opened the pull request:
- The master branch has some conflicts that are too complex to merge at this moment. I will do it later, but this branch should be able to merge.
- The JK's osx builds are ok now at [here](https://travis-ci.org/snoweye/JuniperKernel/builds/322319577) is also built a bit faster.
- The pbdZMQ [master](https://github.com/snoweye/pbdZMQ) should work properly now with the JK's osx builds.
- No need to install otool and install_name_tool. It should have been overwritten from pbdZMQ at [0.3-0](https://github.com/snoweye/pbdZMQ/tree/pbdZMQ_0.3-0) because the `-Wl,-rpath...` seems not overwrite well but took the path from inside `libzmq.*.dylib`.
- `install.libs.R` will need more generalization later for `*` where some osx has 5 instead of 4. Other numbers are expected.

snoweye committed:
Merge pull request #1 from JuniperKernel/master

updates

snoweye committed:
Remove binary files

snoweye committed:
Remove binary files more

snoweye committed:
description updates

snoweye committed:
Manually install pbdZMQ

snoweye committed:
Add repos

snoweye committed:
mk.bat

snoweye committed:
typo

snoweye committed:
Add repo

snoweye committed:
drop devtools

spennihana committed:
Update NEWS.md

spennihana committed:
Revert "Drop binary files" (#11)

* Revert "Update NEWS.md"

This reverts commit 29dc15ef8f23378f415c0f7cba7b5377abe92374.

* Revert "drop devtools"

This reverts commit 07e684b22078a8677eb0c0535e29c731964d8a73.

* Revert "Add repo"

This reverts commit dcfc434d20c7df8b8123a2f82a2c13be24e2019a.

* Revert "typo"

This reverts commit d9b69af7c3441f8edcf249bbf021bcac6b0f8f27.

* Revert "mk.bat"

This reverts commit 64b9271bde7b18ff69fba877252c492465470fdb.

* Revert "Add repos"

This reverts commit 926ae0a5b484a19809d22ba0c496926fe48582d9.

* Revert "Manually install pbdZMQ"

This reverts commit f7ce54a56714b8f074b00fe06e5817a2d896a683.

* Revert "description updates"

This reverts commit f0d118cb0c3beea1122d243dd0d50e6861ed03c4.

* Revert "Remove binary files more"

This reverts commit d4f4b349e152b28b3620908e625f7552bea529a9.

* Revert "Remove binary files"

This reverts commit 824d126e5381ec30c32658c7022d10bcea92c2e8.

snoweye committed:
Add branch install and test

snoweye committed:
disable update-alternative for osx

snoweye committed:
More check

snoweye committed:
Move to Depents

snoweye committed:
remark linux temporary

snoweye committed:
Rollback to import. Try -Wl,-rpath for libzmq.5

snoweye committed:
Build #12 fails on Travis CI???

spennihana committed:
Revert "Revert "Drop binary files" (#11)" (#12)

This reverts commit 52962d9d1260b8177a0e98a5f06d7f1868fcc822.

snoweye committed:
test

snoweye committed:
test

snoweye committed:
test

snoweye committed:
typo

snoweye committed:
Faster Travis

snoweye committed:
test again

snoweye committed:
change travis

snoweye committed:
R CMD check

snoweye committed:
rollback to osx

spennihana committed:
macos to install_name_tool the zmq lib

spennihana commented:
Thanks @snoweye! 

I've re-applied your changes from #10 . I believe that the branch `sp_rpath` (https://github.com/JuniperKernel/JuniperKernel/tree/sp_rpath) works with the current pbdZMQ release.

Would you mind reviewing this branch and perhaps making a PR (in place of this one) with the fixes from `sp_rpath` and any additional changes that you need?

Thanks!

spennihana committed:
rpath problems are fixed, remove otool and install_name_tool changes (#15)

spennihana committed:
update json to latest
update COPYRIGHTS for xtl/xeus/json
update DESCRIPTION with copyright holders and edit description field
remove include/json.hpp and use the nl_json from xeus
force a sleep in the R unit runner

snoweye committed:
merge

snoweye committed:
Merge branch 'JuniperKernel-master'

snoweye commented:
https://travis-ci.org/snoweye/JuniperKernel/builds/323057463
It seems correct on osx with newest pbdZMQ.
Thanks.
